src/components/molecules/ScheduleEvent/ScheduleEvent.scss

```diff
@@ -0,0 +1,99 @@
+@import "scss/constants.scss";
+
+$grey--regular: #3f3d42;
+$grey--hover: #4c494f;
+$grey--users: #4c494f;
+$grey--ligth: #ebebeb;
+
+$purple--regular: #7c46fb;
+$purple--hover: #7c46fb;
+
+$ScheduleEvent--margin-top: 0.25rem;
+$ScheduleEvent--height: 3.75rem;
+$ScheduleEvent--padding: 0 0.25rem 0 0.75rem;
+$ScheduleEvent--border-radius: 18px;
+$ScheduleEvent--box-shadow: 0 5px 10px rgba(0, 0, 0, 0.65);
+
+$bookmark--padding: 0.5rem;
+$bookmark-hover--padding-top: 0.375rem;
+
+$description--margin-top: 2px;
+
+.ScheduleEvent {
+  display: flex;
+  position: absolute;
+  border-radius: $ScheduleEvent--border-radius;
+  padding: $ScheduleEvent--padding;
+  cursor: pointer;
+  height: $ScheduleEvent--height;
+  justify-content: space-between;
+  align-items: center;
+  background-color: $grey--regular;
+  box-shadow: $ScheduleEvent--box-shadow;
+  margin-top: $ScheduleEvent--margin-top;
+  margin-left: var(--event--margin-left);
+  width: var(--event--width);
+
+  &:hover {
+    background-color: $grey--hover;
+    z-index: z(navbar__schedule-event--hover);
+  }
+
+  &--users {
+    color: $grey--hover;
+    background-color: $grey--ligth;
+
+    &:hover {
+      background-color: $white;
+    }
+
+    .ScheduleEvent__bookmark {
+      color: $black;
+    }
+  }
+
+  &--live {
+    background-color: $purple--regular;
+    color: $white;
+
+    &:hover {
+      background-color: $purple--hover;
+    }
+
+    .ScheduleEvent__bookmark {
+      color: $white;
+    }
+  }
+
+  &__info {
+    font-size: 0.8rem;
```

Comment:
[NB] use fontsize from const. 0,8 * 16 =~ 12

Reply:
@mike-lvov Is the const in `px`? Moving to using `em`/`rem` across the codebase is probably a better overall solution for `font-size` than using scss const's I would think?

Reply:
  @0xdevalias I don't mind moving to em/rem eventually. What I want to have is consistency & to achieve it, we should use the defined font-size consts(which are currently defined in px, but can be changed to rem/em in 5 lines of code, which would apply across the app)

Reply:
Yeah, fair enough, makes sense.